cypress.base.config.ts

```diff
@@ -13,7 +12,7 @@ export const baseConfig: Cypress.ConfigOptions = {
     testIsolation: false,
     setupNodeEvents(on, config) {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-var-requires
-      codeCoverage(on, config);
+      // codeCoverage(on, config);
```

Comment:
Lets remove this code all together rather than commenting out.

Reply:
done